Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -5,20 +5,20 @@ import PathKit
 extension XCScheme {
     // swiftlint:disable:next type_body_length
     public final class LaunchAction: SerialAction {
-        public enum Style: String {
+        public enum Style: String, Sendable {
```

Comment:
I'm a bit unfamiliar with the new Swift Concurrency requirements, but why is it required to use `Sendable` here?

Reply:
I believe this is because of `static let defaultLaunchStyle`.  Any static type property must be immutable (`let`) and `Sendable`.